api/proto/info/info.proto

```diff
@@ -201,6 +201,12 @@ enum ZSwState {
   // state. If an App Instance is in ERROR state, it means there is NO DOMAIN
   // currently running.
   ERROR                 = 21;   // Error State
+
+  // It would have been ideal to leave a gap before the ERROR state.
+  // Since we cannot change the existing states now, introduce the
+  // Z_SW_STATE_START_DELAYED state after ERROR.
```

Comment:
We have code which tracks min state in zedmanager and/or volumemgr (mostly about the volumes that go through downloading, verifying, etc) so we can report the minimum in the app instance status. 
I don't think there are other places and states where their numeric values matter. If they do we can have the internal state be in numeric order and have the function which maps to ZSwState take care of it.

Reply:
Yes. I was confused by the types.go values which are different and mapped to the API.